idb/postgres.go

```diff
@@ -58,10 +58,22 @@ type PostgresIndexerDb struct {
 }
 
 func (db *PostgresIndexerDb) init() (err error) {
```

Comment:
I think there are three starting states possible: (has migration state, has accounting state) so follow those; (no migration state, has accounting state) something deployed before now and it will need txid patch; (no migration state, no accounting state) a new deploy after now with no migration needed - no fixes to apply - any schema change would be reflected in the initial schema applied at setup